src/emails/components/ForgotPasswordEmail.tsx

```diff
@@ -0,0 +1,36 @@
+import { Link, Text } from "@react-email/components";
+
+import { EmailLayout } from "./common/EmailLayout";
+
+interface ForgotPasswordEmailProps {
+  firstName: string;
+  token: string;
+}
+
+export default function ForgotPasswordEmail({
+  firstName,
+  token,
+}: ForgotPasswordEmailProps) {
+  return (
+    <EmailLayout>
+      <Text>Hi {firstName},</Text>
+
+      <Text>
+        As you have requested for reset password instructions, here they are,
+        please follow the URL:
+      </Text>
+
+      <Link
+        href={`${!process.env.NEXTAUTH_URL}/reset-password?resetPasswordToken=${token}`}
```

Comment:
environment variables for this project are imported from `@/env`.

can we use APP_URL instead of NEXTAUTH_URL

also the preceding ! shouldn't be used, this turns the url into a truthy / falsy value which is not what we want